lib/Containers/ImmerMemoryPolicy.h

```diff
@@ -0,0 +1,63 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright $YEAR-2021 ArangoDB GmbH, Cologne, Germany
```

Comment:
Let's remove $YEAR here, and insert a copyright header as in the other files.

Reply:
Oops.